### PR TITLE
[Tests-Only] update reva api tests related to sharing

### PR DESCRIPTION
**Pull request by C0rby**
I fixed some issues, unskipped the corresponding tests.
The pr with the fixes:
https://github.com/cs3org/reva/pull/748

Partially fixed
owncloud/ocis-reva#20

Fixed:
owncloud/ocis-reva#26
owncloud/ocis-reva#43
owncloud/ocis-reva#44
owncloud/ocis-reva#94

### CODE REVIEW

**Review by individual-it — CHANGES_REQUESTED** [review]
just a small comment, rest looks good, but please make a PR to ocis and ocis-reva  where this branch is used to run the tests (you just need to change this line in the drone file https://github.com/owncloud/ocis-reva/blob/master/.drone.star#L104 )

**Comment by PVince81** [comment]
there's a conflict

also, should we convert this to a draft PR since the fix itself is not merged, so not ready for review yet?

**Comment by C0rby** [comment]
Yes, I think a draft pr would make sense since we don't know when the reva changes will be merged.
There isn't a way to turn this pr to a draft directly, is it?

**Comment by PVince81** [comment]
it's a bit hard to find :-D
image

File: tests/acceptance/features/apiVersions/fileVersions.feature [hunk 1]
```diff
@@ -67,7 +67,7 @@ Feature: dav-versions
     And user "user0" uploads file "filesForUpload/davtest.txt" asynchronously to "/davtest.txt" in 3 chunks with new chunking and using the WebDAV API
     Then the version folder of file "/davtest.txt" for user "user0" should contain "1" element
 
-  @smokeTest @issue-ocis-reva-94
```
[the remark this answers is not in the document]
Comment (C0rby) [reply tests/acceptance/features/apiVersions/fileVersions.feature:70]: Why should we leave the issue tag?

**Review by individual-it — APPROVED** [review]

**Comment by C0rby** [comment]
@PVince81, I can't merge because codecov/patch is stuck...

**Comment by C0rby** [comment]
Ah nevermind

**Comment by individual-it** [comment]
@C0rby I've faked that codecov CI ;-)

**Pull request merged**